src/components/voter.jsx

```diff
@@ -0,0 +1,74 @@
+import { patchArticleVote } from "../utils/api-calls";
+import up from "../assets/up.png";
+import down from "../assets/down.png";
+import { useContext } from "react";
+import { ArticlesContext } from "../contexts/articles-context";
```

Comment:
This is a choice! Are you planning on using this context in many more components?
Usually, the reason why we want to use Context is because we are using them in many, many components - e.g. a specific Theme (Dark Theme!) which will be used in every component and that spreads across your whole app, not for something that could also be a state.

It's not wrong per se, but it does feel a little bit overkill at the current point. It's also not in line with best practices - have a read on this https://react.dev/learn/passing-data-deeply-with-context#before-you-use-context article of the official react docs, it might help with discerning when you'd want to use context!